src/integration-test/resources/pcw-rules/pcwRuleExample.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please keep the json file's name consistent, name them all by starting with either lower case, or upper case